### PR TITLE
Prepare release v0.22.0

### LINKED ISSUE #638: Bump go version up to latest stable
go1.14 version is out of go support cycle. Also new versions of `spf13` packages use features from later go releases. 

Before release we need to to:
- [ ] update go up to go1.16 or other latest stable version,
- [ ] update `spf13` packages.

Related to #600

### LINKED ISSUE #651: Move all control service requests to `control` subsection
After #628 control service will run on separate from API endpoint. Therefore user won't be able to use the same config file for API and control commands. It will be more clear to have all control service commands in different section. 

Now `netmap snapshot` command uses control service, however other netmap commands are not.

### PR DESCRIPTION
**Pull request by cthulhu-rider**
Closes #638.
Closes #651.

### CODE REVIEW

**Comment by alexvanin** [comment]
Please, update testnet config as well with readme changes.

File: config/testnet/node_config.env [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
-# This is used to open the connection listening socket on your machine
+# This is used to open the connection listening socket(s) on your machine
 #
-#NEOFS_GRPC_ENDPOINT=<BIND_ADDRESS>
-#NEOFS_GRPC_ENDPOINT=1.1.1.1:36512
-NEOFS_GRPC_ENDPOINT=
+#NEOFS_GRPC_NUM=2
+#NEOFS_GRPC_0_ENDPOINT=1.1.1.1:36512
+#NEOFS_GRPC_1_ENDPOINT=2.2.2.2:36513
+NEOFS_GRPC_NUM=
```
Comment (alexvanin) [inline config/testnet/node_config.env:16]: For the testnet I suggest to keep this config as simple as possible and do not describe the possibility of group addresses at all (as we ignore TLS configuration there). The goal is to change config as less as possible for the user.

So I would do something like this.

```
#

# Specify your public IP address or domain name with the open port
# in NODE_ADDRESSES and GRPC_0_ENDPOINT
# By default keep these addresses the same.

# This is used for the public announcement in the network map
#
#NEOFS_NODE_ADDRESSES=1.1.1.1:36512
NEOFS_NODE_ADDRESSES=

# This is used to open the connection listening socket(s) on your machine
#
#NEOFS_GRPC_0_ENDPOINT=1.1.1.1:36512
NEOFS_GRPC_0_ENDPOINT=
NEOFS_GRPC_NUM=1


#https://unece.org/trade/cefact/unlocode-code-list-country-and-territory
# Find suitable LOCODE (from the link above) for your storage node.
#
#NEOFS_NODE_ATTRIBUTE_1=UN-LOCODE:<XX YYY>
#NEOFS_NODE_ATTRIBUTE_1=UN-LOCODE:RU LED
NEOFS_NODE_ATTRIBUTE_1=


NEOFS_NODE_ATTRIBUTE_2=Price:100000
```

**Review by alexvanin — COMMENTED** [review]
Also try to to `make images` before release. I got https://github.com/nspcc-dev/neofs-node/pull/639 issue exactly at the moment I wanted to build image.

File: config/testnet/node_config.env (hunk 1 above)
Comment (cthulhu-rider) [reply config/testnet/node_config.env]: Changed

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,47 @@
...
+### Added
+- Support of Neo wallet credentials in CLI ([#610](https://github.com/nspcc-dev/neofs-node/issues/610)).
+- More reliable approval of trust value by IR ([#500](https://github.com/nspcc-dev/neofs-node/issues/500)).
+- Storage node's ability to announce and serve on multiple network addresses ([#607](https://github.com/nspcc-dev/neofs-node/issues/607)).
+- Validation of network addresses of netmap candidates in IR ([#557](https://github.com/nspcc-dev/neofs-node/issues/557)).
+
+
+### Fixed
+- Approval of objects with with duplicate attribute keys or empty values ([#633](https://github.com/nspcc-dev/neofs-node/issues/633)). 
+- Approval of containers with with duplicate attribute keys or empty values ([#634](https://github.com/nspcc-dev/neofs-node/issues/634)).
```
Comment (alexvanin) [inline CHANGELOG.md:19]: - Default path for CLI config (626)

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,47 @@
 # Changelog
 Changelog for NeoFS Node
 
+## [Unreleased]
+
+## [0.22.0] - 2021-06-29 - Muuido (무의도, 舞衣島)
+
+Storage nodes with a group of network endpoints.
+
+### Added
```
Comment (alexvanin) [inline CHANGELOG.md:10]: - Control service with healthcheck RPC in IR and CLI support (414)

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,47 @@
...
+- More reliable approval of trust value by IR ([#500](https://github.com/nspcc-dev/neofs-node/issues/500)).
+- Storage node's ability to announce and serve on multiple network addresses ([#607](https://github.com/nspcc-dev/neofs-node/issues/607)).
+- Validation of network addresses of netmap candidates in IR ([#557](https://github.com/nspcc-dev/neofs-node/issues/557)).
+
+
+### Fixed
+- Approval of objects with with duplicate attribute keys or empty values ([#633](https://github.com/nspcc-dev/neofs-node/issues/633)). 
+- Approval of containers with with duplicate attribute keys or empty values ([#634](https://github.com/nspcc-dev/neofs-node/issues/634)). 
+
+### Changed
```
Comment (alexvanin) [inline CHANGELOG.md:21]: - IR does not include nodes with LOCODE derived attributes to the network map (412)
- IR uses `morph/client` packages for contract invocations (496)
- Writecache decreases local size when objects are flushed (568)
- IR can override global configuration values only in debug build (363)

**Review by alexvanin — APPROVED** [review]

**Pull request merged**